rpc/api.go

```diff
@@ -126,6 +127,14 @@ func (sc *Client) ClassHashAt(ctx context.Context, address string) (string, erro
 	return *result, nil
 }
 
+// GetStorageAt gets the value of the storage at the given address and key.
+func (sc *Client) GetStorageAt(ctx context.Context, contractAddress, key, blockHashOrTag string) (string, error) {
+	var value string
+	hashKey := fmt.Sprintf("0x%s", caigo.GetSelectorFromName(key).Text(16))
+	err := sc.do(ctx, "starknet_getStorageAt", &value, contractAddress, hashKey, blockHashOrTag)
+	return value, err
```

Comment:
prefer to be explicit with return values. it allows the reader to more confidently make assumptions
```suggestion
		if err := sc.do(ctx, "starknet_getStorageAt", &value, contractAddress, hashKey, blockHashOrTag); err != nil {
		return nil, err
	}

	return value, nil
```